Mchar.gltf

```diff
@@ -0,0 +1,227 @@
+{
+  "accessors": [
+    {
+      "bufferView": 2,
+      "componentType": 5126,
+      "count": 41210,
+      "max": [
+        0.47080400586128235,
+        0.16203799843788147,
+        1.8027470111846924
+      ],
+      "min": [
+        -0.47080901265144348,
+        -0.15409700572490692,
+        -0.0018599999602884054
+      ],
+      "type": "VEC3"
+    },
+    {
+      "bufferView": 2,
+      "byteOffset": 494520,
+      "componentType": 5126,
+      "count": 41210,
+      "max": [
+        0.99997454881668091,
+        0.99997353553771973,
+        0.99993747472763062
+      ],
+      "min": [
+        -0.99997878074645996,
+        -0.99999797344207764,
+        -0.99987733364105225
+      ],
+      "type": "VEC3"
+    },
+    {
+      "bufferView": 3,
+      "componentType": 5126,
+      "count": 41210,
+      "max": [
+        1,
+        0.99982881546020508,
+        0.99999421834945679,
+        1
+      ],
+      "min": [
+        -0.99985408782958984,
+        -0.99983000755310059,
+        -0.99999451637268066,
+        -1
+      ],
+      "type": "VEC4"
+    },
+    {
+      "bufferView": 1,
+      "componentType": 5126,
+      "count": 41210,
+      "max": [
+        0.99480998516082764,
+        0.99739998579025269
+      ],
+      "min": [
+        0.0051799998618662357,
+        0.00095999997574836016
+      ],
+      "type": "VEC2"
+    },
+    {
+      "bufferView": 0,
+      "componentType": 5125,
+      "count": 232296,
+      "max": [
+        41209
+      ],
+      "min": [
+        0
+      ],
+      "type": "SCALAR"
+    }
+  ],
+  "asset": {
+    "extras": {
+      "author": "gome70 (https://sketchfab.com/gome70)",
+      "license": "CC-BY-4.0 (http://creativecommons.org/licenses/by/4.0/)",
+      "source": "https://sketchfab.com/3d-models/male-base-model-c613f570ecda43628018e78ef932b89d",
```

Comment:
This is the author, license, source of the model

Reply:
pretty much yes, it's a screenshot 
just if this style would be convenient or something in lower poly